nodejs/eks/cluster.ts

```diff
@@ -492,6 +492,12 @@ export function createCore(name: string, args: ClusterOptions, parent: pulumi.Co
         data: nodeAccessData,
     }, { parent: parent, provider: provider });
 
+    const oidcProvider = new aws.iam.OpenIdConnectProvider(`${name}-oidcProvider`, {
+      clientIdLists: ["sts.amazonaws.com"],
+      url: eksCluster.identities[0].oidcs[0].issuer,
+      thumbprintLists: ["9E99A48A9960B14926BB7F3B02E22DA2B0AB7280"], // Amazon root CA thumbprint
```

Comment:
I'd suggest pulling this string into a named const that includes a comment linking to an authoritative source. I'm not sure if this value could change in the future, and if so, should be pulled dynamically?

Reply:
Thanks for the PR!

Couple of suggestions:
- Make the creation of the `oidcProvider` optional for users by creating  a new cluster argument for it (e.g. a bool), and in the comments drop in any notes about the functionality and any authoritative sources like AWS docs to learn more.
- As it stands the `oidcProvider` is not being used, nor is it exported in the `CoreData` of the cluster after it's been created. Is this intentional? Should we exporting it? Is there anything else that needs to be done to use this or the creation of this object all that is required?